.github/workflows/enums.yml

```diff
@@ -0,0 +1,103 @@
+name: Custom checks
+
+on:
+  pull_request:
+    branches: [ master ]
```

Comment:
```suggestion
    branches: [ master ]
    paths:
    - 'app/models/**'
```
add a path filter to run only when the content changes?

Reply:
Great suggestion, I'll try it.

Reply:
Works well